src/editor/actions/fetch-cell-parser.js

```diff
@@ -49,7 +46,61 @@ export function emptyLine(line) {
   return line.replace(/\s+/g, "") === "";
 }
 
-export function parseFetchCellLine(line) {
+export function missingFetchType(line) {
+  return !line.trim().match(/^\w+\s*: /);
+}
+
+export function validFetchType(line) {
+  const fetchType = line.trim().split(": ")[0];
+  return fetchType.trimLeft().match(/^(css|js|arrayBuffer|blob|json|text)$/);
```

Comment:
Could we create an array called `VALID_FETCH_TYPES` in this file, and then do something like:

```suggestion
  return VALID_FETCH_TYPES.includes(fetchType.trimLeft());
```